OPERA_RTC-S1_jupyter_book_env.ipynb

```diff
@@ -144,33 +85,34 @@
     "        text=True\n",
     "    )\n",
     "    info = json.loads(conda_info.stdout)\n",
-    "    conda_prefix = Path(f\"{info['root_prefix']}/envs\")"
+    "    conda_prefix = Path(f\"{info['root_prefix']}/envs\")\n",
+    "print(f'conda prefix: {conda_prefix}')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "0de77181-7e0d-4422-9879-1e2f91086c08",
+   "id": "43a017ab-988e-4a3b-93a1-5a8a29ae8dee",
    "metadata": {},
    "source": [
     "---\n",
-    "## 2. Build the `osl_opera_rtc_s1_jbook_env` Conda Environment\n",
+    "## 1. Build the `osl_opera_cslc_jbook_env` Conda Environment\n",
```

Comment:
Should this be the RTC environment, or are we using a shared env for the rtc and cslc notebooks?

Reply:
Good catch!